class/defaults.yml

```diff
@@ -6,14 +6,20 @@ parameters:
         alert-patching.libsonnet: openshift4-monitoring-alert-patching.libsonnet
     namespace: openshift-monitoring
     # TODO: select based on reported OCP version once we have dynamic facts
-    manifests_version: release-4.11
+    manifests_version: release-4.13
     =_cluster_monitoring_operator_version_map:
       release-4.11: release-4.11
+      release-4.12: release-4.13 # TBD: fall back to 4.11 or 4.13?
```

Comment:
Guess we now decided to use 4.13, THis comment can be removed.

Reply:
Updated the comment with more accurate information and findings